app/src/main/java/com/vasilevkin/catsanddogs/features/splash/view/ui/SplashActivity.kt

```diff
@@ -0,0 +1,70 @@
+package com.vasilevkin.catsanddogs.features.splash.view.ui
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.view.Window
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import com.vasilevkin.catsanddogs.R
+import com.vasilevkin.catsanddogs.features.animalList.view.ui.MainActivity
+import com.vasilevkin.catsanddogs.features.splash.ISplashContract
+import org.koin.android.ext.android.inject
+import org.koin.core.parameter.parametersOf
+
+
+/**
+ * Splash Screen with the app icon and name at the center, this is also the launch screen and
+ * opens up in fullscreen mode. Once launched it waits for 2 seconds after which it opens the
+ * MainActivity
+ */
+class SplashActivity : AppCompatActivity(), ISplashContract.View {
+
+    private val splashPresenter: ISplashContract.Presenter by inject { parametersOf(this) }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        makeFullScreen()
+
+        setContentView(R.layout.activity_splash)
```

Comment:
this line of code should be after super.onCreate(savedInstanceState) to avoid mistakes in app working

Reply:
when I do so, it throws an exception:
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.vasilevkin.catsanddogs/com.vasilevkin.catsanddogs.features.splash.view.ui.SplashActivity}: android.util.AndroidRuntimeException: requestFeature() must be called before adding content